README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Vision
 
-This project provides an API for storing and retrieving learning resources that might be helpful to members of [Operation Code](https://operationcode.org/). Ideally, this project will provide the backend for various interfaces for working with the data. The first, and most important front end will be https://operationcode.org/resources.
+This project provides an API for storing and retrieving learning resources that might be helpful to members of [Operation Code](https://operationcode.org/). Ideally, this project will provide the backend for various interfaces for working with the data. The front end can be found at https://operationcode.org/resources.
```

Comment:
You know, having read it again, I'm not sure I like how it flows.
>Ideally, this project will provide the backend for various interfaces for working with the data. The front end can be found at https://operationcode.org/resources.

Maybe it should be something like "The official front end can be found..."? Or perhaps "The main front end is..."?

What do you think @copygodistaken ?

Reply:
I'm not sure I'm as invested as you.  It's a README, and it reads to me just fine!

Reply:
Fair enough. I think people will figure it out 😅